### ISSUE TITLE
Input windows glitches. 

### ISSUE THREAD

**Issue by beeheemooth** [opentoonz/opentoonz#3980]
**Describe the bug**
It looks like entering numbers into input boxes is not working correctly. Watch the video and repeat my steps (do not pay attention to Shift + Esc, this is a shortcut to launch the video recording program).

**To Reproduce**
In the first video, you can see that after pressing Enter, nothing happens, although the number is written in the field. Sometimes numbers are reset to zero. I zeroed the number in the input box after I started dragging the "Accuracy" slider.

![ezgif-2-bc0b9bd9e600](https://user-images.githubusercontent.com/25724133/121817450-37e15700-cc8a-11eb-9d88-85babba42d42.gif)

So what character should you use to enter a fractional number: a comma or a dot? There should be one standard. 

![ezgif-2-ea14c160bcbf](https://user-images.githubusercontent.com/25724133/121817456-3c0d7480-cc8a-11eb-8ab4-deb6377e3d71.gif)

**Device Information:**
 - OS: [e.g. Windows]
 - OpenToonz Version: [e.g. Nightly (2020-01-10)]
 - GPU: [e.g. NVIDIA RTX 2080]
 - GPU Driver Version: [e.g. 440.00]

**Comment by artisteacher** [comment]
Definitely a period for decimals.

**Comment by shun-iwasawa** [comment]
Some locales like German uses comma for a decimal separator. However OT does not accept comma in the input field even if you set the current language to German. OT only accepts period.

In the first video it seems that typing "1,25" or "5,55" does nothing. The brush size is unchanged from zero. In the second video, typing "5,55" in the line width field does not work either. It shows a "rejection" more clearly with red colored flash and resetting the value. 

So, basically they are already under one standard (i.e. they accept period only), but the behaviors are different which may be confusing.

**Comment by beeheemooth** [comment]
@shun-iwasawa The problem is that I can't put a period in the first video and a comma in the second.
In the video below, I write 1.2, but it turns out 12. I'm not talking about the international punctuation standards in fractional numbers, but about the local standard within the program. Why are there different characters for writing a fractional number in different input windows? 
![ezgif-2-38df4a3f73be](https://user-images.githubusercontent.com/25724133/122040135-227e4100-cde0-11eb-90ea-f7248b60fcbf.gif)

**Comment by artisteacher** [comment]
It seems that the style that is active makes a difference. The standard, "no style" brush on a raster level can only use whole numbers - so it ignores the period. The mypaint brushes, on the other hand, do permit decimals.
